compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -691,11 +691,11 @@ object RefChecks {
         case scala.util.Success(v) =>
           ctx.settings.Xmigration.value < v
         case Failure(ex) =>
-          ctx.warning(s"${sym.showLocated} has an unparsable version number: ${ex.getMessage()}", pos)
+          ctx.warning(SymbolHasUnparsableVersionNumber(sym, ex.getMessage()), pos)
           false
       }
       if (changed)
-        ctx.warning(s"${sym.showLocated} has changed semantics in version $symVersion:\n${sym.migrationMessage.get}")
+        ctx.warning(SymbolChangedSemanticsInVersion(sym, symVersion, sym.migrationMessage.get))
```

Comment:
While you're at it. Can you move this into the match. There is no need for this `changed` variable.
Also, `symVersion` is `sym.migrationVersion.get` so you don't need both of them

Reply:
Sure. Updated.